tensorboard/webapp/util/ui_selectors_test.ts

```diff
@@ -234,10 +234,17 @@ describe('ui_selectors test', () => {
 
         expect(getCurrentRouteRunSelection(state)).toEqual(
           new Map([
+            // Inherits false from `selectionState`.
             ['123/run1', false],
+            // legacy name = "apple/run2" and does not match `r.n[1]` and
+            // `apple/r..3`.
             ['123/run2', false],
+            // legacy name = "banana/run1". Inherits true + matches `r.n[1]`.
             ['234/run1', true],
+            // legacy name = "banana/run2". Does not match `r.n[1]` and
+            // `apple/r..3`.
```

Comment:
Consider adding an item that does match apple/r..3 and having its end selection state being true. Otherwise I don't really know if including apple/r..3 in your regex in this test is valuable.